rclcpp/include/rclcpp/any_subscription_callback.hpp

```diff
@@ -214,13 +221,39 @@ class AnySubscriptionCallback
     } else {
       throw std::runtime_error("unexpected message without any callback set");
     }
+    TRACEPOINT(callback_end, (const void *)this);
   }
 
   bool use_take_shared_method()
   {
     return const_shared_ptr_callback_ || const_shared_ptr_with_info_callback_;
   }
 
+  void register_callback_function()
```

Comment:
@iluetkeb nit: in the context of an `AnyCallback` method, wouldn't it make more sense for this method to be called `register_tracepoints()` or something along those lines?

Reply:
registering tracepoints is an actual thing with LTTng/tracers, but at this point I doubt it would cause confusion

Reply:
I think we could apply the same rationale as below on this one i.e. make the name be related to tracing in some way.